src/main/asciidoc/inc/image/_naming.adoc

```diff
@@ -50,6 +50,9 @@ When specifying the container name pattern the following placeholders can be use
 | *%i*
 | An index which is incremented if a container has already been created. With this parameter it is easily possible to have multiple, similar containers. See the example below for more details.
 
+| *%r*
```

Comment:
```suggestion
| *%e*
```

Can we use something different than `%r` please ? Because in the future we might also to add `%r` as a placeholder for a client-side random number, that we add to some pattern.